DbgEngWrapper/DbgEngWrapper.h

```diff
@@ -2767,6 +2767,12 @@ namespace DbgEngWrapper
             [In] ULONG BytesRequested,
             [In] BYTE* buffer,
             [Out] ULONG% BytesRead);
+		
+		generic <typename TValue>
```

Comment:
Could you please add a comment above this explaining that it's a convenience wrapper around `ReadVirtual`?

Reply:
Added